test/unit/blocks_sensing.js

```diff
@@ -229,6 +229,33 @@ test('loud? boolean', t => {
     t.end();
 });
 
+test('get attribute of sprite variable', t => {
+    const rt = new Runtime();
+    const sensing = new Sensing(rt);
+    const s = new Sprite();
+    const target = new RenderedTarget(s, rt);
+    const thing = {
+        name: 'cars',
+        value: 'trucks'
+    };
+    rt.getSpriteTargetByName = () => target;
+    target.lookupVariableByNameAndType = () => thing;
+    t.equal(sensing.getAttributeOf({PROPERTY: 'cars'}), 'trucks');
+
+    t.end();
+});
+test('get attribute of variable that does not exist', t => {
+    const rt = new Runtime();
+    const sensing = new Sensing(rt);
+    const s = new Sprite();
+    const stage = new RenderedTarget(s, rt);
+    rt.getTargetForStage = () => stage;
+    stage.lookupVariableByNameAndType = () => null;
+    t.equal(sensing.getAttributeOf({PROPERTY: 'stage'}), 0);
```

Comment:
The non-existent variable should have a less confusing name because it's not immediately clear what's happening here.

Reply:
Agreed. Mostly copy/paste error.